backend/users/views.py

```diff
@@ -277,27 +274,18 @@ class HighLevelAgencyAPI(APIView):
     permission_classes = [IsAuthenticated]
 
     def post(self, request):
-        try:
-            data = request.data
-            high_level_instance = HighLevelModel.objects.update_or_create(
-                user_id=request.user.id,
-                defaults=data
-            )
-            return Response(
+        data = request.data
+        HighLevelModel.objects.update_or_create(
+            user_id=request.user.id,
+            defaults=data
+        )
+        agency_api_key = request.data['agency_api_key']
+        data = get_celery_task(agency_api_key)
```

Comment:
This is a wrong way of calling celery task, It shoudl be called using delay method